src/mesh-analysis.rs

```diff
@@ -0,0 +1,17 @@
+use nalgebra as na;
+use nalgebra::base::Vector3;
+use nalgebra::geometry::Point3;
+use std::collections::HashSet;
```

Comment:
We usually split imports into groups as follows:

1) `pub` imports (none here) 
2) std imports, if any (`std::collections::HashSet` in this case)
3) external crate imports, if any (all `nalgebra` imports here)
4) `crate` imports, if any (`crate::geometry::{Geometry, NormalStrategy, TriangleFace}` import here)
5) `super` imports

... and separate the groups by a newline. This way, the most local import group is last and the most global is first. Concretely in this case, it would be:

```rust
use std::collections::HashSet;

use nalgebra as na;
use nalgebra::base::Vector3;
use nalgebra::geometry::Point3;

use crate::geometry::{Geometry, NormalStrategy, TriangleFace};
```

Reply:
understood